src/sentry/integrations/vsts/issues.py

```diff
@@ -31,14 +31,15 @@ def get_create_issue_config(self, group, **kwargs):
         project_choices = []
         initial_project = ('', '')
         for project in projects:
-            project_choices.append((project['id'], project['name']))
-            if project['id'] == self.default_project:
+            project_id_and_name = '%s#%s' % (project['id'], project['name'])
+            project_choices.append((project_id_and_name, project['name']))
+            # TODO(lb): Properly handle default project after it has been implemented.
+            if project_id_and_name == self.default_project:
                 initial_project = project['name']
         return [
             {
                 'name': 'project',
```

Comment:
This is an asside and not part of this PR, but shouldn't this be capital?

Reply:
`'name'` you mean? I kind of assume the frontend takes care of how things are displayed and I'm just providing the content. But if they need to be capitalized on the backend do let me know.